src/integration-test/tests/citizen/offers/offers_test.ts

```diff
@@ -32,3 +32,26 @@ Scenario('I can as a defendant make an offer, accept offer and counter sign the
   offerSteps.viewClaimFromDashboard(claimRef)
   I.see('You’ve both signed a legal agreement. The claim is now settled.')
 })
+
+Scenario('I can make an offer as a defendant to a claimant and have the claimant reject it @citizen', function* (I: I) {
+  const claimantEmail: string = yield I.createCitizenUser()
+  const defendantEmail: string = yield I.createCitizenUser()
+
+  const claimRef: string = yield I.createClaim(createClaimData(PartyType.INDIVIDUAL, PartyType.INDIVIDUAL), claimantEmail)
+
+  I.linkDefendantToClaim(claimRef, claimantEmail, defendantEmail)
+  I.respondToClaim(claimRef, claimantEmail, createResponseData(PartyType.INDIVIDUAL), defendantEmail)
+
+  userSteps.login(defendantEmail)
+  offerSteps.makeOfferFromDashboard(claimRef)
+  I.see('Your offer has been sent to ' + createClaimant(PartyType.INDIVIDUAL).name)
```

Comment:
I guess you can extract claimant name form result of `createClaimData(PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)` (line 40). Doing this twice is a risk that somebody creates a claim with individuals and then expects text for some other type of party.

Extract result of `createClaimData(PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)` to variable